src/Controllers/RegisterController.php

```diff
@@ -66,6 +66,11 @@ public function registerAction(): RedirectResponse
 
         $user->fill($this->request->getPost($allowedPostFields));
 
+        // Workaround for email only registration/login
+        if ($user->username === null) {
+            $user->username = null;
+        }
```

Comment:
We should definitely think about a better solution down the road, this is pretty hacky 😅
Alternate syntax:
```suggestion
        $user->username ??= null;
```

Reply:
After all, the both codes are the same.
Both are funny!

This is a temporary hack, so no problem.

Reply:
Since the username column is unique, why don't use email value as username for temporary hack?

Reply:
I think we should not store data other than what the column name indicates.
It will only bring confusion.

Reply:
Yes, but it can only save one (1) row ATM.

Reply:
This is a modification a developer could make for their instance, but using an email address as the username can be a security and privacy issue so this library has worked hard to keep them separate.